src/parser/utils.ts

```diff
@@ -1,3 +1,14 @@
+import { Buffer } from 'buffer';
+export type BufferEncoding =
```

Comment:
is this really not provided by the node or `buffer` types?

Reply:
It's there but its not exported, maybe we should just accept string, it would mean not having to ever change this if there was some new format. (waiting on utf32 to take off 😆)